### PR TITLE
3 setup circleci be

### PR DESCRIPTION
**Pull request by sicknacco**
## Description
This PR contains the work for CircleCi setup. config file created and one test written for implementation.

## Related Issue
[Link to any related issue(s) that this PR addresses, e.g., "Closes #123"]

## Checklist
- [x] I have reviewed the code changes.
- [x] I have tested this code.
- [x] I have requested an update to the documentation.
- [x] I have checked for and resolved any merge conflicts.
- [x] I have assigned reviewers for this PR.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    docker: # run the steps with Docker
+      - image: cimg/ruby:3.2.2 # circleci/ruby:2.7.2 # if you're using ruby version >= 3.0, use cimg/ruby:3.x.y 
+        environment: # environment variables for primary container
+          BUNDLE_JOBS: 3
+          BUNDLE_RETRY: 3
+          BUNDLE_PATH: vendor/bundle
+          PGHOST: 127.0.0.1
+          PGUSER: postgres
+          RAILS_ENV: test
+      - image: circleci/postgres:11 # database image
```
Comment (topher-nullset) [inline .circleci/config.yml:16]: interesting, why does 11 work and not 14... given we use postgres14

File: spec/models/character_spec.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+require 'rails_helper'
+
+RSpec.describe Character, type: :model do
```
Comment (topher-nullset) [inline spec/models/character_spec.rb:3]: testy testy...

File: .circleci/config.yml (hunk 1 above)
Comment (brad2412) [inline .circleci/config.yml:16]: Nice job trying postgress: 14.  we will have to look into why that part wasn't working and where.

**Review by topher-nullset — APPROVED** [review]
Nice setup!! CircleCI sure makes this SO much easier lol.

**Review by brad2412 — APPROVED** [review]

**Pull request merged**